packages/ActionMenu/src/ActionMenu.tsx

```diff
@@ -0,0 +1,253 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import List, { OptionType, Option } from '@igloo-ui/list';
+import Dropdown, { Position, ReferenceProps } from '@igloo-ui/dropdown';
+import IconButton from '@igloo-ui/icon-button';
+import Kebab from '@igloo-ui/icons/dist/Kebab';
+
+import './action-menu.scss';
+
+export enum Keys {
+  Enter = 'Enter',
+  Space = ' ',
+  ArrowDown = 'ArrowDown',
+  ArrowUp = 'ArrowUp',
+  Escape = 'Escape',
+  Tab = 'Tab',
+  Home = 'Home',
+  End = 'End',
+}
+
+export type FocusDirection = 'first' | 'last' | 'up' | 'down';
+
+export type ActionMenuOption = Omit<Option, 'type'>;
+
+export interface ActionMenuProps extends React.ComponentProps<'div'> {
+  /** Add a specific class to the action menu */
+  className?: string;
+  /** Whether or not the action menu should close when an option is selected */
+  closeOnSelect?: boolean | ((option: OptionType) => boolean);
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Whether or not the action menu should be open by default */
+  isOpen?: boolean;
+  /** Callback when the action menu is closed  */
+  onMenuClose?: () => void;
+  /** Callback when the action menu is opened  */
+  onMenuOpen?: () => void;
+  /** Callback when an option is selected */
+  onOptionSelect?: (option: OptionType) => void;
+  /** A list of options to display in the action menu */
+  options: ActionMenuOption[];
+  /** Position of the action menu. */
+  position?: Position;
+}
+
+const ActionMenu: React.FunctionComponent<ActionMenuProps> = (
+  props: ActionMenuProps
+) => {
+  const {
+    className,
+    closeOnSelect = true,
+    dataTest,
+    isOpen = false,
+    onMenuClose,
+    onMenuOpen,
+    onOptionSelect,
+    options,
+    position = 'bottom-start',
```

Comment:
by default the position must be `bottom-end`
```suggestion
    position = 'bottom-end',
```